app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split()
+
+    if len(command) != 3:
+        return
+
+    if command[0] != "mv":
+        return
+
+    if command[1] == command[2]:
+        return
+
+    second_file_dir = "/".join(command[2].split("/")[:-1])
```

Comment:
Do not use any specific directory delimiters, your code should be platform-agnostic